salt/cloud/clouds/hetzner.py

```diff
@@ -289,7 +329,10 @@ def create(vm_):
     )
 
     # Get the ssh_keys
-    ssh_keys = vm_.get("ssh_keys", None)
+    ssh_keys = config.get_cloud_config_value(
+        "ssh_keys", vm_, __opts__, search_global=False, default=False
```

Comment:
Why are you setting the default to be False here? It looks like it should stay set to None.

Reply:
Good question, I think I got inspired by [the digitalocean driver](https://github.com/saltstack/salt/blob/master/salt/cloud/clouds/digitalocean.py#L323) here, but `None` seems adequate, indeed.